<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workUi</class>
 <widget class="QWidget" name="workUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Header</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="dateEdit">
        <property name="currentSection">
         <enum>QDateTimeEdit::DaySection</enum>
        </property>
        <property name="displayFormat">
         <string>d/M/yy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="pushSave">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="../../../core/res.qrc">
          <normaloff>:/conferma24x24.png</normaloff>:/conferma24x24.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Comanda</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,1,0" columnminimumwidth="0,0,0,0">
      <item row="0" column="0" colspan="4">
       <widget class="QTableView" name="tableComanda"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Amount Comanda</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLCDNumber" name="lcdComanda">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>$</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Places Comanda</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QSpinBox" name="spinLocalPlaces"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushNew">
        <property name="text">
         <string>Insert Article</string>
        </property>
        <property name="icon">
         <iconset resource="../../../core/res.qrc">
          <normaloff>:/aggiungi16x16.png</normaloff>:/aggiungi16x16.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QPushButton" name="pushAdd">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset resource="../../../core/res.qrc">
          <normaloff>:/rigth_arrow-32.png</normaloff>:/rigth_arrow-32.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Everything</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1,0">
      <item row="0" column="0" colspan="3">
       <widget class="QTableView" name="tableEverything"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Amount Everything</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="lcdEverything">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>$</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Total places</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QSpinBox" name="spinPlaces"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushDelete">
        <property name="text">
         <string>Delete Article</string>
        </property>
        <property name="icon">
         <iconset resource="../../../core/res.qrc">
          <normaloff>:/elimina24x24.png</normaloff>:/elimina24x24.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../core/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushAdd</sender>
   <signal>clicked()</signal>
   <receiver>workUi</receiver>
   <slot>addComanda()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushDelete</sender>
   <signal>clicked()</signal>
   <receiver>workUi</receiver>
   <slot>deleteRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushNew</sender>
   <signal>clicked()</signal>
   <receiver>workUi</receiver>
   <slot>newRowPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>deleteRow()</slot>
  <slot>save()</slot>
  <slot>noSave()</slot>
  <slot>addComanda()</slot>
  <slot>newRowPressed()</slot>
  <slot>dateChanged(QDate)</slot>
 </slots>
</ui>
